sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -1060,6 +1062,29 @@ SUBROUTINE CLDRAD
       ENDIF  
 !
 !***  BLOCK 2.  2-D CLOUD FIELDS.
+
+! GSD maximum cloud fraction in (PBL + 1 km) (J. Kenyon, 8 Aug 2019)
```

Comment:
@jaymes-kenyon and @hu5970 , I have been working on setting up new baselines for hrrr and rap. It turns out cloud base height and pressure (ID 711 and 798) are not reproducible with different tasks in both hrrr and rap tests. Do you think anything is missing or incorrect in my syncing version?

Reply:
Hi Wen.  This is surprising, and I'm not yet sure on the reason.  One possibility:  in GSD's HRRR runs on Jet, we experienced occasional UPP crashes, and we found that lowering the fill value for the CLDZ(i ,j) array fixed this.  For example, after CLDZ is passed to GRID1, we added:  

GRID1(I,J) = max(min(GRID1(I,J), 20000.0),0.0)

This change was only made in GSD's UPP master branch on VLab, so it's not in your synched code.  

Another possibility is that the cloud base height/pressure algorithm has a bug.  It uses a horizontal search of nearby grid points (the "ceil_neighbor") array, which might not be parallelizing correctly.  I'm ready to help if we believe it's a bug.